src/components/commentsList/CommentsList.tsx

```diff
@@ -1,24 +1,26 @@
-import type { Comment as IComment } from '../../types';
+import { nanoid } from 'nanoid';
 import Comment from '../comment/Comment';
+import { classes } from './CommentsList.st.css';
+import type { Comment as IComment } from '../../types';
 
 interface ICommentListProps {
     comments: IComment[];
-    likeComment: () => void;
+    likeComment: (date: number, userName: string) => void;
 }
 
 const CommentsList = ({ comments, likeComment }: ICommentListProps) => (
-    <ul>
-        {comments.map(
-            ({ user: { userName, profilePicture }, creationDate, content, LikesCount }) => (
-                <Comment
-                    user={{ userName, profilePicture }}
-                    creationDate={creationDate}
-                    content={content}
-                    LikesCount={LikesCount}
-                    likeComment={likeComment}
-                />
-            )
-        )}
+    <ul className={classes.root}>
+        {comments.map(({ user: { userName, profilePicture }, creationDate, content, LikedBy }) => (
+            <Comment
+                user={{ userName, profilePicture }}
+                creationDate={creationDate}
+                content={content}
+                LikedBy={LikedBy}
+                likeComment={likeComment}
+                key={nanoid()}
```

Comment:
Call this 'uid; or 'id'instead of key.

Reply:
"key" is a saved keyword [https://reactjs.org/docs/lists-and-keys.html#keys:~:text=it%20on%20CodePen-,Keys,-Keys%20help%20React](url)

Reply:
add ID to comment type and make this his key